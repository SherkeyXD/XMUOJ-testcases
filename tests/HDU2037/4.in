88
9 21
6 8
14 22
17 23
15 23
0 8
5 15
16 23
19 22
5 6
15 19
14 16
4 12
21 22
10 20
2 5
15 22
5 13
17 19
0 18
0 15
13 21
4 23
4 22
2 12
21 22
12 14
11 18
16 18
13 21
8 13
17 19
2 19
5 23
9 11
12 20
15 17
21 22
0 7
2 9
16 22
6 16
21 23
6 20
18 21
1 12
4 8
8 10
10 23
21 22
3 17
1 18
8 23
9 14
5 9
2 20
1 9
8 14
14 19
4 19
1 18
14 21
9 21
9 14
21 22
3 9
22 23
2 7
18 22
1 20
9 20
11 21
8 11
17 22
7 19
9 11
19 22
22 23
1 3
20 21
2 17
20 22
9 13
15 19
0 15
18 21
7 11
19 22
59
5 10
15 16
14 19
9 11
19 22
21 22
8 17
9 21
20 21
17 19
0 2
21 23
10 20
10 18
0 14
5 6
3 11
12 20
10 22
14 19
18 19
8 10
14 17
9 19
14 21
8 19
6 9
22 23
15 19
19 21
7 11
10 13
17 23
10 18
9 10
10 17
6 18
10 14
4 11
0 18
9 12
20 22
19 23
16 18
6 16
0 8
8 23
12 18
9 21
9 18
0 2
12 14
7 20
3 15
4 15
3 21
2 19
20 23
0 13
30
1 20
16 19
9 10
18 21
7 17
17 20
3 10
11 13
11 17
3 22
12 21
2 17
16 21
8 18
0 2
13 14
16 21
18 23
19 23
17 23
13 22
15 22
20 22
19 23
4 19
0 5
1 20
18 20
11 15
16 20
64
3 21
22 23
10 21
6 17
19 23
15 20
17 21
20 22
7 18
4 20
5 18
9 10
7 18
20 23
3 14
4 19
7 18
4 12
9 20
0 21
7 8
19 23
17 20
3 8
1 10
11 15
10 17
22 23
4 22
9 11
19 20
10 13
3 11
19 23
16 19
13 15
1 9
2 9
1 12
19 21
9 22
19 23
17 21
6 21
10 13
1 12
12 13
7 8
13 17
17 20
22 23
22 23
20 22
13 21
8 17
12 23
14 19
13 18
6 20
5 16
12 23
6 14
20 22
5 22
42
19 22
6 19
19 23
9 22
14 15
15 19
22 23
21 22
9 11
11 16
14 20
9 22
2 18
7 23
19 22
18 19
17 19
20 22
21 23
11 18
5 18
14 15
3 8
9 10
0 11
1 22
2 20
17 19
3 8
17 21
20 23
13 22
4 13
2 19
10 19
7 22
12 21
8 19
17 22
11 12
3 4
21 23
67
5 8
2 7
8 20
15 19
3 22
9 16
17 20
22 23
8 17
1 6
9 19
7 22
22 23
6 22
19 21
8 18
3 13
20 22
16 21
17 21
15 20
18 21
22 23
10 20
1 18
8 23
7 22
3 22
5 18
10 23
10 23
9 13
0 12
4 10
16 20
18 20
16 18
5 9
9 19
3 9
2 19
21 22
8 17
16 22
6 22
22 23
1 17
8 16
9 14
22 23
4 8
19 21
6 23
13 23
5 20
22 23
2 6
16 18
12 14
8 14
8 21
11 20
21 23
4 22
1 13
9 13
6 9
49
6 13
18 21
7 22
11 16
2 5
14 20
13 21
3 18
10 22
4 15
3 14
3 20
9 14
16 21
19 20
7 17
13 20
11 21
16 20
4 14
22 23
12 16
8 19
7 21
9 16
15 22
12 23
1 13
15 17
20 21
9 16
0 19
5 23
1 17
0 13
17 23
8 10
19 23
9 21
0 20
16 19
2 8
12 15
21 22
10 14
8 9
6 23
7 18
14 15
56
4 23
18 19
12 15
5 9
9 14
1 2
21 22
17 23
2 11
3 18
22 23
22 23
20 21
10 21
20 22
4 21
7 10
11 16
4 22
22 23
10 21
12 19
15 19
21 22
2 17
3 18
20 23
10 16
0 14
2 20
9 21
15 18
13 22
12 18
20 22
13 21
19 20
18 19
8 17
3 4
8 22
17 23
15 23
17 20
8 10
5 23
15 17
17 21
18 19
7 21
0 13
3 17
10 11
22 23
10 17
11 15
93
4 6
16 19
5 12
1 2
3 6
9 23
15 19
0 14
22 23
2 18
14 21
17 21
22 23
17 23
15 18
12 14
7 20
20 23
8 23
7 11
4 6
11 14
6 11
8 11
17 18
13 17
12 22
15 20
3 20
16 20
13 21
15 23
9 10
16 20
8 14
2 15
18 23
7 13
11 12
21 22
3 13
13 15
17 18
7 10
16 20
13 15
7 23
21 23
21 22
14 23
20 21
15 17
2 18
0 10
16 18
15 19
20 23
11 13
13 23
17 18
7 23
7 8
13 16
20 21
0 18
16 23
15 20
13 22
10 16
5 20
22 23
1 17
9 10
21 23
4 23
19 20
15 23
10 22
18 21
21 23
2 19
11 18
8 19
6 13
11 14
19 20
8 14
12 17
4 13
1 16
20 23
17 18
12 17
37
16 23
3 22
11 16
12 19
16 23
12 20
10 14
11 13
4 23
19 20
12 18
22 23
6 22
11 20
1 17
17 18
14 17
22 23
21 22
16 18
0 11
8 22
6 23
11 21
11 16
16 21
13 23
15 18
17 21
18 22
8 12
2 13
22 23
14 19
22 23
5 7
21 23
0
