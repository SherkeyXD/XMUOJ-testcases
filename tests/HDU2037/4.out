10
11
6
8
7
7
9
11
12
7
